docs/source/index.rst

```diff
@@ -24,21 +24,15 @@ Welcome to the Cozmo SDK Beta!
 
 .. toctree::
     :maxdepth: 2
-    :caption: Get Started
-
-    getstarted.rst
-
-.. toctree::
-    :maxdepth: 2
     :caption: Downloads
 
     downloads.rst
 
 .. toctree::
     :maxdepth: 2
-    :caption: Beginner Tutorial
+    :caption: Get Started
 
-    tutorial-beginner.rst
```

Comment:
Thought I removed all references to it. Seems here the error is that tutorial-beginner.rst is still in your build docs/source. I deleted it from my version of that folder, I'm unclear on what causes tutorial-beginner.rst to be present in your build